packages/a11y-testing/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@fluentui/a11y-testing",
+  "version": "0.1.0",
+  "description": "Fluent UI tests for a11y patterns.",
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "sideEffects": [
+    "lib/version.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "just-scripts build",
+    "bundle": "just-scripts bundle",
+    "clean": "just-scripts clean",
+    "code-style": "just-scripts code-style",
+    "just": "just-scripts",
+    "lint": "just-scripts lint",
+    "start": "just-scripts dev:storybook",
+    "start-test": "just-scripts jest-watch",
+    "test": "just-scripts test",
+    "update-api": "just-scripts update-api",
+    "update-snapshots": "just-scripts jest -u"
+  },
+  "devDependencies": {
+    "@fluentui/common-styles": "^0.2.2",
+    "@fluentui/eslint-plugin": "^0.53.3",
+    "@fluentui/react-conformance": "^0.1.0",
+    "@fluentui/storybook": "^0.2.6",
+    "@testing-library/react": "^10.4.9",
```

Comment:
I don't think we are using this anymore, as far as I remember it was only used in the `HooksTestFacade`, but we decided to go only with the `ComponentsTestFacade`

Reply:
removed